dask/array/tests/test_fft.py

```diff
@@ -27,9 +31,21 @@ def key(k):
     "ihfft",
 ]
 
+all_nd_funcnames = [
+    "fft2",
+    "ifft2",
+    "fftn",
+    "ifftn",
+    "rfft2",
+    "irfft2",
+    "rfftn",
+    "irfftn",
+]
+
 nparr = np.arange(100).reshape(10, 10)
 darr = da.from_array(nparr, chunks=(1, 10))
 darr2 = da.from_array(nparr, chunks=(10, 1))
+darr3 = da.from_array(nparr, chunks=(10, 10))
```

Comment:
Fixed a bug with the `irfft[2n]` case.

However, I'm still seeing something strange with the `rfft[2n]` case. FWICT this is a bug or at a bare minimum an inconsistency between the transform's and inverse transform's shape. Have opened an issue with NumPy. More details in that issue.

xref: https://github.com/numpy/numpy/issues/8855